eac2277.md

```diff
@@ -0,0 +1,17 @@
+# Open Source Development
+
+Name: Erick Cifuentes
+
+UNI: eac2277
+
+Github: [link](https://github.com/ErickCif)
+
+GitHub README: [link](https://github.com/ErickCif/ErickCif/blob/main/README.md)
```

Comment:
this should be in HW1

Reply:
Right, I didn't know if you wanted two separate pull requests on these changes, sorry about that! Is this okay how it is or would you like me to do something different?